app-forensics/rkhunter/Manifest

```diff
@@ -1,2 +1,8 @@
-DIST rkhunter-1.4.2.tar.gz 277707 BLAKE2B 97968801ce5dccefc7d5eb246fd8031091447b9090c5bc9b958ea64835c47661d1f40b5a6352e0c92608a514f237ac2946a7f84498915418581a71d16b497f61 SHA512 a4e45caaaf5b8262619ebb890784c75c4e30db4c6c0eba305f86d419142b4796c95bc55fe8846dce8d58bc7636bdb365a4a8c41707f64d4d81373687c5a3b0d4
-DIST rkhunter-1.4.4.tar.gz 297626 BLAKE2B 9d9a50e3541817d8dfec6c76665ae84b92f49ed3128244cb5359f2149203ccdc7cdaa05db3da87f6aa6ee79fa84491f40cdc3443d77359b64c23947cb685516e SHA512 87e9c617220765678cc4519eee27d1d56185c3a7fb1d6338c8fb984ac4f5176c31bb54b69e1de615d66a0cf1e72b672e66b368e37851a459def69463cbb8661e
+AUX rkhunter-1.3.cron 3927 SHA256 525b2680a025eb612600aadb52047718ecfd198b012ab938da150c143cc374e0 SHA512 a99eb535c12b6715031aa367ba0ae70ed1761a03b74c57d173512006e466ef56bfc7ffb15364d4f36d39597277a8cf3b35e7286462bfd6ec9d44877dd1653d4d
+AUX rkhunter-1.4.2.conf.patch 1250 SHA256 ce828b367b04286e5ca9b564455bfadbb9cdfe01dfb0d6601f45adb49265142c SHA512 654887bea5d3b7388c11c9a59d3d924d763268c3552bee82a0bcf9b5077803eaa0cf16b7346aeaac44e817bfdd30e5d74abd10d12eb07f0bc1a97c57b850aefe
+AUX rkhunter.bash-completion 2499 SHA256 cdd08c1e6b79d2875d88c35d0c0bf9d85d84b4a1b21b97eb208883466d1efd39 SHA512 8fba540f5425169810b8baaf2e801638e3286aa93a69d4a96f5916e3bc9b632edaef6b509b8ca58f8047a7c7d4bb4bf348ad7923e855edce3dd0432725a944e7
+DIST rkhunter-1.4.2.tar.gz 277707 SHA256 789cc84a21faf669da81e648eead2e62654cfbe0b2d927119d8b1e55b22b65c3 SHA512 a4e45caaaf5b8262619ebb890784c75c4e30db4c6c0eba305f86d419142b4796c95bc55fe8846dce8d58bc7636bdb365a4a8c41707f64d4d81373687c5a3b0d4
+DIST rkhunter-1.4.4.tar.gz 297626 SHA256 a8807c83f9f325312df05aa215fa75ad697c7a16163175363c2066baa26dda77 SHA512 87e9c617220765678cc4519eee27d1d56185c3a7fb1d6338c8fb984ac4f5176c31bb54b69e1de615d66a0cf1e72b672e66b368e37851a459def69463cbb8661e
+EBUILD rkhunter-1.4.2.ebuild 1411 SHA256 d02e361bcd6e16ec86e351a307156b7c779bb8d684129791beeeef5771b77492 SHA512 e4b2f70f6417f3a44d918ab310460dd45f7fae760458f7b9804752a79beffabea832dd195c85efeb8e31d1b71d7196b1df70a8cdc5a562d0a4ef575888820ada
+EBUILD rkhunter-1.4.4.ebuild 1462 SHA256 cfa28389e0fd0153b79b35d5bcbf7d6ae004a6328e71b9e41d2f64ee6a11bbd1 SHA512 8e0de63eff6d0933000d22a207ea3db1e161db6c864853c5dfe7201d2a406a67b6cfcf3342fc6e55cf39a7ae077421d6bfce01407c8228c3cb4e9fc5e7e87009
+MISC metadata.xml 250 SHA256 b7958445feebc53822c26943827589ae43981f57398e4907fd2f2b176f34b17b SHA512 ab31238a72b35a1f33056cf1736628bbf367fc20f4dc93735d11617ed668f295567e1865844d727e666de07c56d84f8b5cfd1eec41c91cd836789a420ea2dda9
```

Comment:
Oh no!

Looks like you had something change for the metadata/layout.conf because this is going backwards.